### PR TITLE
[DPE-4794] Update bundle with all components in the mysql ecosystem + add smoke test

### PR DESCRIPTION
**Pull request by shayancanonical**
## Issue
1. We would like to update our bundle with all possible charms in the mysql ecosystem. This will help us be able to gather all artifacts needed to deploy the ecosystem in an airgapped environment
2. We would like to add a test that will let us know when a component of the bundle will error out

## Solution
1. Update bundle
2. Add smoke test

### CODE REVIEW

File: tox.ini
Comment (dragomirp) [inline tox.ini:1]: Going forward that can be moved to poetry and reusable workflows.

**Comment by dragomirp** [comment]
> question: how will this bundle be used? I'm a bit confused about having two data integrators and a mysql-test-app and worried that someone might deploy this bundle as-is
> 
> also, should mysql-test-app or sysbench be deployed in a production environment?

The purpose is to get all related components for airgapped environments and to have a demo with everything included. Production envs seem to mostly use their own bespoke bundles.

**Review by dragomirp — APPROVED** [review]

**Comment by dragomirp** [comment]
> i'm a bit worried that this might cause mistakes—e.g. I don't think users in airgapped environments will or should deploy mysql-test-app

If airgapped users don't need it, it will just remain in cache in the store proxy. If they end up needing it and it's not in cache, it will need to be added separately. Either way, adding it was requested for the Postgresql bundle, I would imagine the same applies for the Mysql bundle.

File: releases/latest/mysql-bundle.yaml [hunk 1]
```diff
@@ -1,29 +1,142 @@
...
+    channel: latest/edge
+    charm: data-integrator
+    constraints: arch=amd64
+    num_units: 1
+    revision: 27
+    series: jammy
+    to:
+    - '0'
+  grafana-agent:
+    channel: latest/edge
```
Comment (paulomach) [inline releases/latest/mysql-bundle.yaml:12]: question: do we want to test against all things `edge` or only the ones under the team umbrella?

**Review by paulomach — APPROVED** [review]

File: releases/latest/mysql-bundle.yaml (hunk 1 above)
Comment (shayancanonical) [reply releases/latest/mysql-bundle.yaml:12]: i will merge as is, as postgresql has `edge` for charms not in the data platform umbrella. however, i will confirm with @taurus-forever and open a follow up PR if necessary

File: releases/latest/mysql-bundle.yaml [hunk 2]
```diff
@@ -1,29 +1,142 @@
 applications:
+  data-integrator:
+    channel: latest/edge
+    charm: data-integrator
+    constraints: arch=amd64
```
[the remark this answers is not in the document]
Comment (shayancanonical) [reply releases/latest/mysql-bundle.yaml:5]: may not be strictly necessary, but i guess i opted for explicit-ness and consistency with the postgresql bundle

File: releases/latest/mysql-bundle.yaml (hunk 1 above)
Comment (shayancanonical) [reply releases/latest/mysql-bundle.yaml:12]: alex's answer: we should use edge for now, but there will be more changes upcoming to this bundle in the near future

**Pull request merged**